config.toml

```diff
@@ -92,16 +106,23 @@ algolia_docsearch = false
 
 #Enable offline search with Lunr.js
 offlineSearch = true
+offlineSearchSummaryLength = 70
+offlineSearchMaxResults = 10
 
-github_branch= "main"
+# Enable syntax highlighting and copy buttons on code blocks with Prism
+prism_syntax_highlighting = true
 
 # User interface configuration
 [params.ui]
 # Enable to show the side bar menu in its compact state.
-sidebar_menu_compact = true
+sidebar_menu_compact = false
```

Comment:
I tend to prefer having things compacted, and letting users open what they want/need and not confusing them with a too long list

Reply:
This only keeps open the path from what you're reading to the root. It makes extremely hard to read documentation about a topics that is spread in sibling files (on the same level), eventually having to jump to one level up to a page that is not in the path to the root. This documentation site is not simple enough or organized well enough to make this kind of reading a good experience. Or at least this is my opinion so far. We can put it back, if you wish.

Reply:
Not to mention, I had to test that when the left-side navigation tree does not fit on the page, it is placed and scrolls properly.

Reply:
OK fine.